plugins/rule_engines/irods_rule_engine_plugin-irods_rule_language/functions.cpp

```diff
@@ -1,5 +1,8 @@
 /* For copyright information please refer to files in the COPYRIGHT directory
  */
+
+#define MAKE_IRODS_ERROR_MAP
+#include "rodsErrorTable.h"
```

Comment:
i'd move this under the other includes so that they aren't affected by the macro.  you could also undef it following the include.  this is more of a recommendation than anything.

Reply:
I tried moving it under the other includes, but the namespaces I needed were then no longer visible.  I could still do it but it would require making a new compilation unit just for the new microservice.

Reply:
that is likely due to the other headers including rodsErrorTable.h.  carry on :)